src/datastore/postgres-store.ts

```diff
@@ -862,6 +905,23 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
     return { results: txIds };
   }
 
+  async getBlockTxsRows(indexBlockHash: string) {
```

Comment:
This should be named `blockHash` to match what is queried below. The `index_block_hash` is distinct from this.